scripts/make_flavors.rb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/ruby
```

Comment:
Looks like a great start. Rather than `/scripts/make_flavors`, it would probably make sense to add this as a rake task instead. I think that is more in fitting with normal Ruby paradigms anyway. Does that make sense?

Reply:
That does make sense, but I'll need to read up on the "proper way" of doing this. First attempt at ruby.

Do you see the rake task being ran as part of the build process, or as part of the installation process on the host?

Reply:
No worries, feel free to ask questions as you have them. I appreciate you working through this with me. I would imagine it would happen on the development side and/or as part of the build process, rather than during installation. So the release would have the output bundled with it, to keep things relatively simple (though it would require a new release to add now info, this is the state we are already in).